modules/doc/config.yml

```diff
@@ -37,9 +37,30 @@ Extensions:
         repo: https://github.com/idaholab/moose
         google-cse: 000781241768298771085:pedtnlxdkx8
         menu:
-            Getting Started: getting_started.menu.md
-            Documentation: documentation.menu.md
-            Help: help.menu.md
+            Getting Started:
+                Installing MOOSE: getting_started/installation/index.md
+                Creating an Application: getting_started/new_users.md
+                Tutorials: getting_started/examples_and_tutorials/index.md
+                Learning Resources: getting_started/learning_resources/index.md
+            Documentation:
+                Application Usage: application_usage/index.md
+                Application Development: application_development/index.md
+                Framework Development: framework_development/index.md
+                Modules: modules/index.md
```

Comment:
```suggestion
                Physics Modules: modules/index.md
```

Reply:
Yeah, I thought about that name too. There's just those few modules that aren't physics-related, so I wasn't sure of the best name.

Reply:
I'd call these all non-physics:

- functional expansion tools
- misc
- rdg
- reactor
- stochastic tools
- xfem